cloud/scope/clients.go

```diff
@@ -138,7 +145,7 @@ func (c *ClientProvider) createClients(region string) (OCIClients, error) {
 	if err != nil {
 		return OCIClients{}, err
 	}
-	identityClient, err := c.createIdentityClient(region, c.ociAuthConfigProvider, c.Logger)
+	identityClt, err := c.createIdentityClient(region, c.ociAuthConfigProvider, c.Logger)
```

Comment:
The name change was because their is a package `identityClient` of the same name.